pennylane/math/multi_dispatch.py

```diff
@@ -241,24 +241,40 @@ def concatenate(values, axis=0, like=None):
     if like == "torch":
         import torch
 
-        device = (
-            "cuda"
-            if any(t.device.type == "cuda" for t in values if isinstance(t, torch.Tensor))
-            else "cpu"
-        )
+        device_set = set()
```

Comment:
`codecov` is failing! Is there any way to check multi-GPU on PL CI, or should we just add `pragma: no cover`?

Reply:
have you tried the `gpu` label? Is it helping any way?